src/charts/helpers/configuration.test.js

```diff
@@ -0,0 +1,15 @@
+import { applyConfiguration } from './configuration';
+
+describe('Configuration', () => {
+    it('should allow setting empty string', () => {
+        const expected = '';
+        const mockChart = {
+            'expected': (value) => {
+                mockChart._expected = value;
+            },
+        };
+        let actual = applyConfiguration(mockChart, { expected: '' })._expected;
+
+        expect(actual).toEqual(expected);
+    });
+});
```

Comment:
Awesome!

Reply:
Thanks, looking forward using your react bridge and contributing more if I can

Reply:
@Golodhros are you planing a new release soon?

Reply:
I can do it today!

Reply:
Awesome!! Thanks. @dalerasrorov-eb @Golodhroa